src/main/java/fly/wild/wizards/serverside/ServerConnector.java

```diff
@@ -0,0 +1,328 @@
+package fly.wild.wizards.serverside;
+
+import java.io.IOException;
+import java.time.LocalTime;
+import java.util.UUID;
+import org.jboss.as.controller.client.ModelControllerClient;
+import org.jboss.dmr.ModelNode;
+import org.jboss.as.cli.ControllerAddress;
+import org.jboss.as.cli.Util;
+import org.jboss.as.cli.impl.ModelControllerClientFactory;
+import org.jboss.as.cli.operation.OperationFormatException;
+import org.jboss.as.cli.operation.impl.DefaultOperationRequestBuilder;
+
+import fly.wild.wizards.tlswizard.controller.OneWayTLSConfigurationConfiguration;
+import fly.wild.wizards.tlswizard.controller.TLSConfiguration;
+
+public class ServerConnector {
+
+	private String ipAddress;	
+	private String distinguishedName;
+	private String alias;
+	private StringBuilder logMessage;
+	private final String STRING_CLEAR_TEXT_VALUE_KEY_STORE = "keystorepass";
+	private final String STRING_ALGORITHM_VALUE = "RSA";
```

Comment:
Perhaps we can take a look at this later? For example, if we add fields in GUI to configure the keystore password, it might be handy to have it here so that we can update it in the constructor. WDYT?

Reply:
@theashiot Sounds good!